src/drive/web/modules/views/Recent/index.spec.jsx

```diff
@@ -96,5 +100,11 @@ describe('Recent View', () => {
     expect(
       parentDiv1.getElementsByClassName('fil-file-path')[0].textContent
     ).toEqual(path)
+
+    // navigates  to the history view
+    const historyItem = getByText('History')
+    fireEvent.click(historyItem)
+
+    waitFor(() => expect(getByText('Current version')).toBeTruthy())
```

Comment:
I don't think you need to `waitFor` no? 

`getByText` waits by default. Something like 

```
expect(() => {
getByText()
}).not.toThrow();
```

should be enough

Reply:
That was my first attempt, but it doens't work. I also don't think `getByText` waits, that's `findByText`. I'm not sure of the difference between `findByText`  and `waitFor(getByText)` though.

Reply:
Cool I'll use findBy then since it's the same.

Event hough the `expect` is not needed, I find it good to add them as it makes it explicit that the line is really testing something, and not a forgotten line.